### ISSUE TITLE
Redesign the Font View

### ISSUE THREAD

**Issue by davelab6** [fontforge/fontforge#72]
Other font editors present the glyph table in their Font Views in a structured way, which helps users understand and navigate a character set on more than just an individual glyph by glyph way. The proportions of the cells can also be redesigned to be more pleasing.

**Comment by davelab6** [comment]
@andreaslarsen you'll like this issue :)

**Comment by andreaslarsen** [comment]
yep, I think one of my first comments was on this topic - font-view is currently not very noob friendly. Bookmarked.
